layouts/services/single.html

```diff
@@ -36,7 +36,9 @@ <h6 class="with-dash white">
         <h2>{{- .title -}}</h2>
         <div class="row middle-xs between-xs logos">
             {{- range .logos -}}
-                {{- partial (printf "icons/%s.html" .icon) -}}
+            <div class="col-xs-6 col-sm-4 col-md-2">
```

Comment:
Same problem as on home page

Reply:
But without it, they aren't displace evenly? What would be the best way to fix it?

Reply:
You can see [here](https://github.com/inBeat/inbeat-website/blob/e2af8beb791d9fd7bcba53cf8ecd074d1b5e7291/themes/inbeat/layouts/index.html#L17-L26) and [here](https://github.com/inBeat/inbeat-website/blob/e2af8beb791d9fd7bcba53cf8ecd074d1b5e7291/themes/inbeat/assets/scss/pages/_home.scss#L51-L81) what I did for inBeat :)